tests/test_views.py

```diff
@@ -107,7 +121,11 @@ def test_select2_view_text_poll_content_repr(self):
                 data={"content_type_id": self.poll_content_contenttype_id},
             )
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.json()["results"][0]["text"], str(self.poll_content))
+        expected_result_list = [
+            {"text": "example", "id": 1},
+            {"text": "example2", "id": 2},
+        ]
+        self.assertListEqual(response.json()["results"], expected_result_list)
```

Comment:
So lets say the json looks like this:
```
{
    'results': [...],
    'blablabla': 'cats',
}
```
Is it correct for the test to still pass? (it would) I would have thought that no.